multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -37,7 +37,12 @@ coreos:
       enable: true
       content: |
         [Service]
-        ExecStart=/usr/bin/kubelet \
+        ExecStartPre=/usr/bin/mkdir -p /var/log/containers
+        ExecStartPre=/usr/bin/mkdir -p /var/log/fluentd-position
+        Environment="RKT_OPTS=--volume var-log-fluentd-position,kind=host,source=/var/log/fluentd-position --mount volume=var-log-fluentd-position,target=/var/log/fluentd-position --volume var-log-containers,kind=host,source=/var/log/containers --mount volume=var-log-containers,target=/var/log/containers"
```

Comment:
discussed in meatspace. Lets' back this out for now and follow up mounting /var/log/containers in a follow-up PR